.github/workflows/trivy.yml

```diff
@@ -0,0 +1,18 @@
+name: build
+on:
+  pull_request:
+jobs:
+  build:
+    name: Trivy Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Run Trivy vulnerability scanner in Image mode
+        uses: aquasecurity/trivy-action@0.28.0
+        with:
+          scan-type: 'fs'
+          format: 'table'
+          exit-code: '1'
+          trivyignores: '.github/.trivyignore'
```

Comment:
also ich weiß nicht, ob der `.github` Ordner der richtige ist. ich hätte es eher eine Ebene drüber gemacht. Vorallem, wenn wir das auch in Gitlab einführen wollen